Tests/SwiftSyntaxMacroExpansionTest/BodyMacroTests.swift

```diff
@@ -178,7 +178,7 @@ final class BodyMacroTests: XCTestCase {
           } else {
             statements
           }
-        return body.map(\.self)
+        return body.map({ $0.self })
```

Comment:
Do you think `Array(body)` would convey the intention better than mapping to `self`?

Reply:
I hadn't thought of that, it's better indeed. I will update it.